.github/workflows/pull-compliance-docs.yml

```diff
@@ -2,16 +2,30 @@ name: compliance-docs
 
 on:
   pull_request:
-    paths:
-      - "docs/**"
-      - "*.md"
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 
 jobs:
+  files-changed:
+    name: detect what files changed
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    # Map a step output to a job output
+    outputs:
+      docs: ${{ steps.changes.outputs.docs }}
+    steps:
+      - uses: actions/checkout@v3  # v2
+      - name: Check for backend file changes
+        uses: dorny/paths-filter@v2  # v2
+        id: changes
+        with:
+          filters: .github/file-filters.yml
```

Comment:
Could even extract the whole block to a file.

https://docs.github.com/en/actions/using-workflows/reusing-workflows#example-reusable-workflow